src/Catty/Embroidery/EmbroideryDSTService.swift

```diff
@@ -117,7 +117,10 @@ class EmbroideryDSTService: EmbroideryProtocol {
     }
 
     func byteFromConversionTable(position: Int) -> UInt {
-        position < 0 ?
+        guard position > -121 && position < 121 else {
```

Comment:
Could you please add constants with proper names instead of using `121` and `-121 `(in the whole file) to give me and others a little bit more context of what those values refer to? Thanks!

Reply:
Sorry, that really was ugly. Resolved.